### PR TITLE
DOC: stats: Some small corrections to a couple docstrings.

### PR DESCRIPTION
**Pull request by WarrenWeckesser**
Fix a couple minor issues that made it past code review in their pull requests.

### CODE REVIEW

File: scipy/stats/stats.py [hunk 1]
```diff
@@ -7350,9 +7350,9 @@ def rankdata(a, method='average', *, axis=None):
     >>> rankdata([[0, 2], [3, 2]]).reshape(2,2)
     array([[1. , 2.5],
           [4. , 2.5]])
-    >>> rankdata([[0, 2], [3, 2]], axis = 1)
-    array([[1., 2.],
-          [2., 1.]])
+    >>> rankdata([[0, 2, 2], [3, 2, 5]], axis=1)
+    array([[1. , 2.5, 2.5],
+           [2. , 1. , 3. ]])
```
Comment (tylerjereddy) [inline scipy/stats/stats.py:7355]: Are those spaces after the decimal points from the NumPy repr? I assume that's not formally pep8?

Comment (WarrenWeckesser) [reply scipy/stats/stats.py:7355]: Yes, that's the output generated by numpy, which is not something we should edit.

**Comment by WarrenWeckesser** [comment]
I noticed another trivial issue in the `f_oneway` docstring, so I included the fix here.

**Review by tylerjereddy — APPROVED** [review]
Minor docstring adjustments and CI is green, so merging thanks.

**Comment by mdhaber** [comment]
Thanks @WarrenWeckesser

**Pull request merged**